sdks/python/setup.py

```diff
@@ -87,6 +87,7 @@ def get_version():
     'avro>=1.7.7,<2.0.0',
     'dill>=0.2.5,<0.3',
     'google-apitools>=0.5.2,<1.0.0',
+    'googledatastore==6.3.0',
```

Comment:
Could we pin this similar to the other things here, with <7.0.0. Do you know what is the backward compatibility policy for googledatastore?